specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimanagement.json

```diff
@@ -78,6 +78,10 @@
             "$ref": "#/definitions/ErrorFieldContract"
           },
           "description": "The list of invalid fields send in request, in case of validation error."
+        },
```

Comment:
Question. Typically details is an array of ErrorResponseBody (see common-type/resource-manager/v1/types.json). It come thru well in SDK such as for .NET as Exception.InnerExcpetions. Why intorudce another object as innerError like this?

Reply:
Thanks, reverted this change. We will fix this in the service.